### DISCUSSION TITLE
Como fazer backup do Ubuntu e passar para o Arch Linux?

### DISCUSSION THREAD

**Discussion by rafgam10** [frontendbr/forum#2370] [Perguntas e Respostas]
Quero mudar de distro para o Arch Linux, mas não quero perder meu dados do Ubuntu tem como fazer backup do Ubuntu e usar no Arch Linux? me ajudar pls!

**Comment by ninetails** [comment]
Assim, dados vc diz?

Se são projetos, tento deixá-los sempre no github, qualquer coisa é só mudar de máquina e baixar o projeto novamente, já que Linux eu uso somente para programar e tal.

Mas creio que você pode tentar copiando toda a sua pasta home (`/home/**seu_user**/`) pra máquina nova e depois arrumar o que for incompatível e instalar o que precisar e tal.

Ou tentar gerenciar seus `dotfiles`, num repositório privado (ou público, lembrando de não botar credenciais).

Mas creio que tirando configurações, vc consegue passar tudo via _oldstyle_: pendrive/hd externo ou serviço de storage em cloud (Google Drive, OneDrive, Dropbox etc)...

Só que programas, creio que seria difícil você fazer backup, o ideal seria reinstalar dado que apesar de ambos usarem o mesmo Kernel do Linux, o Ubuntu tem um gerenciador de pacotes baseado no Debian e o Arch Linux, no Pacman com um repositório de pacotes mantido por usuários (AUR). Talvez teria a facilidade de gerenciar alguns pacotes instalados pelo Snap, que não se altera...... mas usar o AUR facilita muito caso migre pro Arch :)

**Reply by rafgam10** [reply]
tem algum vídeo para me ajudar a fazer isso??

**Reply by ninetails** [reply]
Sobre `.dotfiles` eu cheguei a fazer um tempo o que foi explicado [neste vídeo (em inglês, desculpe)](https://youtu.be/awtfkl50bUQ). Talvez você consiga algum vídeo em português.

E enviar pra um pendrive/drive online, creio que com qualquer navegador de arquivos visual vc consiga copiar arrastando (claro, dependendo do que for copiar), como usando o Nautilus do Ubuntu ou o comando `cp` do Linux.

Tem alguma dúvida em específico sobre qual forma de backup?
